pynamodb/attributes.py

```diff
@@ -501,33 +508,114 @@ def deserialize(self, value):
 
 class UTCDateTimeAttribute(Attribute):
     """
-    An attribute for storing a UTC Datetime
+    A UTC DateTime attribute
+
+    :param ttl: If True, this attribute is used as the TTL for purposes of automatic item expiry and deletion
+
+        Only a single attribute in a table may be used as the TTL. The first attribute with TTL enabled
+        will be used; repeated use of the flag will be ignored.
+
+        Note that expired items may persist up to two days after their TTL expires. See: 
+        https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_UpdateTimeToLive.html
+    """
+    attr_type = NUMBER
```

Comment:
This change will break the existing API - any existing models using UTCDateTimeAttribute will be expecting the ISO-8601 formatted strings.  Couldn't we add this as a TTLDateTimeAttribute or EpochTimestampAttribute instead to be more explicit about the new functionality without breaking existing model behavior?

Reply:
Like the Boolean update, it reads both the old and new formats, but writes the new one. The opposite behavior (reading both formats but writing the old one) is available from LegacyUTCDateTimeAttribute. I pretty much just copied the work that was done for the boolean switch.

Reply:
The boolean update could not be safely deployed. See https://github.com/pynamodb/PynamoDB/issues/404

Reply:
@jpinner-lyft #404 is closed, does that mean that this is ready?